Code/Tools/SceneAPI/SceneData/Behaviors/ImportGroup.h

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#pragma once
+
+#include <SceneAPI/SceneCore/Components/BehaviorComponent.h>
+#include <SceneAPI/SceneCore/Events/ManifestMetaInfoBus.h>
+#include <SceneAPI/SceneCore/Events/AssetImportRequest.h>
+
+namespace AZ::SceneAPI::Behaviors
+{
+    class ImportGroup 
```

Comment:
Can this be final?

Reply:
um, probably, I don't think I have a compelling reason for it not to be. If i need to restart my AR for any reason I'll change it.